components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/configuration/models/ServiceDiscoveryConfigurations.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.apimgt.core.configuration.models;
+
+import org.wso2.carbon.kernel.annotations.Configuration;
+import org.wso2.carbon.kernel.annotations.Element;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to hold Service Discovery configurations and Generate YAML file
+ */
+@Configuration(namespace = "wso2.carbon.apimgt.serviceDiscovery",
```

Comment:
can you check whether it is acceptable to use camel case namespace entries please?

Reply:
I checked in here [1] and in the code comments. They had not mention anything specifically. But I found the namespace given for SecureVault as "wso2.securevault" [1]. Therefore I'll also change mine to "wso2.carbon.apimgt.servicediscovery". Thanks for noticing.

[1] https://github.com/wso2/carbon-config/blob/master/docs/UpdatingConfigurations.md